libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -455,6 +455,68 @@ AP_GPS_UBLOX::_request_next_config(void)
 #endif
         break;
 
+    case STEP_F9: {
+        if (_hardware_generation == UBLOX_F9 && params.gnss_mode != 0) {
+
+            AP_GPS_UBLOX::config_list gnss_cfg[16];
```

Comment:
should 16 be UBLOX_MAX_GNSS_CONFIG_BLOCKS?

Reply:
Fixed